masterlist.yaml

```
@@ -8790,12 +8745,7 @@ plugins:
       - Graphics
   - name: 'SAFO - Valdacil.esp'
     msg:
-      - type: warn
-        content:
-          - lang: en
-            text: 'This plugin seems to be outdated and may not be compatible with the latest version of its master files.'
-          - lang: de
-            text: 'Dieses Plugin scheint veraltet zu sein und ist möglicherweise nicht mehr kompatibel mit der aktuellsten Version seiner Master-Dateien.'
+      - <<: &obsolete
```

This message implies that there's an updated version which I don't believe there is. `patchOutdated` would be more appropriate.

```suggestion
      - <<: &patchOutdated
```